cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -19,6 +19,7 @@ import (
 	kcpkg "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/packaging/v1alpha1"
 	kcclient "github.com/vmware-tanzu/carvel-kapp-controller/pkg/client/clientset/versioned"
 	versions "github.com/vmware-tanzu/carvel-vendir/pkg/vendir/versions/v1alpha1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: lets have a more explicit name
```suggestion
	corev1 "k8s.io/api/core/v1"
```

Reply:
Updated